README.md

```diff
@@ -219,6 +219,21 @@ A configuration that can be passed to the [bell][bell] OAuth module to authentic
 #### Expected Promise Response
 1. Resolve with a valid [bell][bell] configuration
 
+### getWebhookStrategy
+
+#### Expected Outcome
+A hapi webhook auth strategy object that can be passed to the hapi server.
```

Comment:
So the parseHook() should  take care of the auth for the webhook? Then we need to make it clear in the base that the parseHook is responsible for that. 
Can I import and re-use this? https://github.com/mhazy/hapi-github-webhooks/blob/master/lib/hmac.js#L11-L17